spec-main/guest-window-manager-spec.ts

```diff
@@ -33,7 +33,11 @@ describe('new-window event', () => {
     proxy: {
       snapshotFileName: 'proxy-window-open.snapshot.txt',
       browserWindowOptions: {
-        show: false
+        show: false,
```

Comment:
Why did this have to be changed?

Reply:
because the test was wrong before. it was supposed to be testing the "proxy", i.e. `nativeWindowOpen: false`, but had not been updated to cope with the new default of nativeWindowOpen.